lib/components-schema-v1_0_x.ts

```diff
@@ -95,29 +99,57 @@ export const componentsDefinitionSchema_v1_0_x = {
                                     options: {
                                         type: 'array',
                                         minItems: 1,
+                                        items: {
+                                            type: 'object',
+                                            properties: {
+                                                caption: {
```

Comment:
Can you add descriptions for the new properties added?

Reply:
sure